doc/man1/openssl-s_client.pod.in

```diff
@@ -656,7 +656,11 @@ For example strings, see L<SSL_CTX_set1_sigalgs(3)>
 =item B<-curves> I<curvelist>
 
 Specifies the list of supported curves to be sent by the client. The curve is
-ultimately selected by the server. For a list of all curves, use:
+ultimately selected by the server.
+
+The list of all supported curves includes EC with parameters as well as X25519
+and X448, which do not support EC parameters. For a list of EC that support
+parameters, use:
```

Comment:
This is misleading.

```suggestion
The list of all supported curves includes named EC parameters as well as X25519
and X448 or FFDHE groups. For a list of named EC parameters, use:
```

Reply:
I like it. The use of "named EC parameters" for Weierstrass curves is much less ambiguous.